dcos/mesos.py

```diff
@@ -133,7 +133,8 @@ def slave_file_read(self, slave_id, path, offset, length):
         url = self.slave_url(slave_id, 'files/read.json')
         params = {'path': path,
                   'length': length,
-                  'offset': offset}
+                  'offset': offset,
+                  'timeout': self._timeout}
```

Comment:
I think we shouldn't include this in `params`. Pass it as a regular function argument.